util/cron/nightly

```diff
@@ -633,7 +633,7 @@ if ($examples == 1) {
 if ($blogonly == 1 || ($testblog == 1 && $examples == 1)) {
     # test the sub-directories explicitly to avoid a bunch of skipped directories,
     # and also avoid the problem of testing the archetypes directory
-    $testdirs .= " $blogdir/chpl-src/ $blogdir/content/posts";
+    $testdirs .= " $testdir/copied-chapel-blog/";
```

Comment:
Should this be `copied-chapel-blog/chpl-src/` and `copied-chapel-blog/content/posts` separately so it's the explicit subdirectories as before?

Reply:
it should be OK this way because the only thing we copied into the `$testdir` were the desired directories.

Reply:
I see. In that case I think the comment above this line should be updated or removed since we don't have to limit to subdirectories here anymore